client/odysseus/message/index.tsx

```diff
@@ -0,0 +1,74 @@
+import classnames from 'classnames';
+import { RefObject, useState } from 'react';
```

Comment:
Might be a good idea to separate type imports from JS module imports - I always recommend using `import type` when we can.